### ISSUE TITLE
[BUG] Updating ReactiveUI.Events.XamForms to 9.17.1 fails and prevents the project from compiling

### ISSUE THREAD

**Issue by AndreasHennig** [reactiveui/ReactiveUI#2073]
**Describe the bug**

When I try to upgrade the ReactiveUI.Events.XamForms package from 9.16.6 to 9.17.1 it breaks my project. The package upgrade failes giving the following error:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error		PharmacistNuGetTask: System.ArgumentNullException: Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   at Pharmacist.Core.Generation.ReflectionExtensions.GetRealType(IType type, ICompilation compilation) in d:\a\1\s\src\Pharmacist.Core\Generation\ReflectionExtensions.cs:line 126
   at Pharmacist.Core.Generation.ReflectionExtensions.GetEventType(IEvent eventDetails) in d:\a\1\s\src\Pharmacist.Core\Generation\ReflectionExtensions.cs:line 98
   at Pharmacist.Core.Generation.Resolvers.EventNamespaceResolverBase.IsValidParameters(IEvent eventDetails) in d:\a\1\s\src\Pharmacist.Core\Generation\Resolvers\EventNamespaceResolverBase.cs:line 68
   at Pharmacist.Core.Generation.Resolvers.PublicEventNamespaceResolver.<>c.<GetValidEventDetails>b__0_0(IEvent x) in d:\a\1\s\src\Pharmacist.Core\Generation\Resolvers\PublicEventNamespaceResolver.cs:line 22
   at System.Linq.Enumerable.WhereArrayIterator`1.MoveNext()
   at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   at System.Linq.OrderedEnumerable`1.<GetEnumerator>d__1.MoveNext()
   at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at Pharmacist.Core.Generation.Generators.InstanceEventGenerator.GenerateEventWrapperClass(ITypeDefinition typeDefinition, IEnumerable`1 events) in d:\a\1\s\src\Pharmacist.Core\Generation\Generators\InstanceEventGenerator.cs:line 99
   at Pharmacist.Core.Generation.Generators.InstanceEventGenerator.<>c.<Generate>b__1_3(ValueTuple`2 x) in d:\a\1\s\src\Pharmacist.Core\Generation\Generators\InstanceEventGenerator.cs:line 33
   at System.Linq.Enumerable.WhereSelectListIterator`2.MoveNext()
   at System.Linq.Enumerable.WhereEnumerableIterator`1.MoveNext()
   at System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   at Pharmacist.Core.Generation.Generators.InstanceEventGenerator.<Generate>d__1.MoveNext() in d:\a\1\s\src\Pharmacist.Core\Generation\Generators\InstanceEventGenerator.cs:line 33
   at System.Linq.Enumerable.<SelectManyIterator>d__17`2.MoveNext()
   at Microsoft.CodeAnalysis.SyntaxList`1.CreateNode(IEnumerable`1 nodes)
   at Pharmacist.Core.ObservablesForEventGenerator.<ExtractEventsFromAssemblies>d__5.MoveNext() in d:\a\1\s\src\Pharmacist.Core\ObservablesForEventGenerator.cs:line 122
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Pharmacist.Core.ObservablesForEventGenerator.<ExtractEventsFromNuGetPackages>d__4.MoveNext() in d:\a\1\s\src\Pharmacist.Core\ObservablesForEventGenerator.cs:line 108
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Pharmacist.MsBuild.PharmacistNuGetTask.Execute() in d:\a\1\s\src\Pharmacist.MSBuild\PharmacistNuGetTask.cs:line 100	PlanningPoker.UnitTests	C:\Users\andy_\.nuget\packages\pharmacist.msbuild\1.0.95\buildTransitive\netstandard2.0\Pharmacist.MSBuild.targets	46	
```

If I try to build the project for Android afterwards the same erros occurs.

**Steps To Reproduce**
1. Go to 'Nuget Package Manager'
2. Click on 'Manage Nuget Packages for Solution'
3. Update the ReactiveUI.Events.XamForms package
4. The mentioned error occurs

**Expected behavior**
The package should be upgraded normally to version 9.17.1 and the project should compile and work on Android and iOS as it did before.

**Environment**

- OS:       Windows 10
- Version   Visual Studio 2019, Version 16.1.3

**Comment by glennawatson** [comment]
Thanks for the bug report. This should be a easy one to fix and will do it later today.

**Comment by glennawatson** [comment]
Out of curiosity can you attach a copy of the project references in your project?

**Comment by glennawatson** [comment]
Fixed in 9.17.4
